### PR TITLE
Tournaments: Add viewbanlist

### PR DESCRIPTION
**Pull request by Rhythms88**

### CODE REVIEW

**Comment by Rhythms88** [comment]
done

File: tournaments/index.js [hunk 1]
```diff
@@ -973,6 +973,13 @@ let commands = {
 		acceptchallenge: function (tournament, user) {
 			tournament.acceptChallenge(user, this);
 		},
+		viewbanlist: function (tournament, user) {
```
Comment (sirDonovan) [inline tournaments/index.js]: The second argument isn't needed.

File: tournaments/index.js [hunk 2]
```diff
@@ -973,6 +973,13 @@ let commands = {
...
+			if (!this.runBroadcast()) return;
+			if (tournament.banlist.length < 1) {
+				return this.errorReply("The tournament's banlist is empty.");
+			}
+			this.sendReplyBox("<b>Banlist:</b><br />" + Chat.escapeHTML(tournament.banlist.join(', ')));
```
Comment (sirDonovan) [inline tournaments/index.js]: I'd change this to <code>This tournament's banlist:</code>.

**Comment by Rhythms88** [comment]
sigh, did,

**Pull request merged**